chain/messagepool/repub.go

```diff
@@ -91,7 +98,7 @@ func (mp *MessagePool) republishPendingMessages() error {
 
 		// we don't republish negative performing chains, as they won't be included in
 		// a block anyway
-		if chain.gasPerf < 0 {
+		if !allowNegative && chain.gasPerf < 0 {
```

Comment:
As a note, we should republish slightly negative chains. Base fee can fall quite quickly.

Reply:
I didn't even think about this effect. In effect, miners who updated to 0.5.10 won't re-pub messages.

Reply:
Not sure about that, we might as well wait for the next republish interval.

Reply:
But yeah, the effect is real -- and I didn't even think about it either.